client/my-sites/woocommerce/woop/style.scss

```diff
@@ -22,78 +22,92 @@ body.is-section-woocommerce-installation.theme-default.color-scheme {
 	.woocommerce_landing-page-empty-content {
 		padding-top: 3em;
 		max-width: 540px;
+		margin: 0 auto;
+
+		@media ( max-width: calc( 540px + 15px + 15px ) ) {
+			margin: 0 15px;
+		}
 
 		.empty-content__title {
 			@include onboarding-heading-text;
 			color: var( --studio-gray-90 );
 			padding: 1em 0 0.5em;
 		}
+
 		.empty-content__line {
 			@include onboarding-large-text;
 			margin-bottom: 1.5em;
 		}
 	}
-}
-
-.woop__landing-page-features-section {
-	@include woocommerce-colors();
-	margin-top: 100px;
-	padding-top: 60px;
-	padding-bottom: 40px;
-	padding-left: 20px;
-    padding-right: 20px;
-
-	// on desktop make the section full width so it takes the whole space
-	@media ( min-width: 660px ) {
-		margin-left: -30px;
-		width: calc( 100% + 20px );
-	}
 
-	background-color: var( --color-woocommerce-onboarding-background );
-	.formatted-header .formatted-header__title {
+	.woop__landing-page-features-section {
+		@include woocommerce-colors();
+		background-color: var( --color-woocommerce-onboarding-background );
+		margin-top: 100px;
+		padding-top: 60px;
+		padding-bottom: 40px;
 
-		@media ( max-width: $break-mobile ) {
-			margin-left: 0;
-			margin-right: 0;
-			width: 100%;
-			max-width: fit-content;
-			text-align: center;
+		.formatted-header .formatted-header__title {
+			@include onboarding-font-recoleta;
+			text-align: left;
+			font-size: 2rem;
+			max-width: 377px;
+			line-height: 40px;
+			padding-left: 0; // Match promo-card pad-left
 		}
 
-		text-align: left;
-		margin-left: 32px;
-		font-size: 2rem;
-		max-width: 377px;
-		line-height: 40px;
-		@include onboarding-font-recoleta;
-	}
+		.promo-section {
+			.promo-section__promos .promo-card {
+				box-shadow: none;
+				background-color: var( --color-woocommerce-onboarding-background );
 
-	.promo-section__promos .promo-card {
-		// 3 column layout
-		@media ( min-width: $break-wide ) {
-			width: calc( 33% - 1em );
-		}
-		box-shadow: none;
-		background-color: var( --color-woocommerce-onboarding-background );
+				.action-panel__title {
+					justify-content: normal;
+					font-weight: 500;
+					font-size: $font-body;
 
-		.action-panel__title {
-			@media ( max-width: $break-mobile ) {
-				display: block;
-				width: 100%;
-				text-align: center;
+					.gridicon {
+						padding-right: 0.5rem;
+						fill: #0675c4; // TODO: maybe we need to define this color as a variable somewhere?
+					}
+				}
+
+				.action-panel__body {
+					font-size: $font-body-small;
+				}
 			}
-			font-weight: 500;
-			font-size: 1.25rem;
 		}
 
-		.action-panel__body {
-			@media ( max-width: $break-mobile ) {
-				text-align: center;
-			}
+		// Extend grey background outside the bottom stripe
+		&::before {
+			content: '';
+			background-color: var( --color-woocommerce-onboarding-background );
+			position: absolute;
+			height: 100%;
+			width: 200vw;
+			left: -100vw;
+			z-index: -1;
+			margin-top: -60px;
 		}
```

Comment:
This `::before` hack replaces the need to do a negative margin and a >100 width which was making it hard to deal with the column layout (it also needs to work with the width / margins to make sure the feature content aligns with the formatted header.